libs/components/forms/src/lib/modules/error/errors.component.ts

```diff
@@ -0,0 +1,56 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { ValidationErrors } from '@angular/forms';
+import { SkyIdModule } from '@skyux/core';
+import { SkyI18nModule } from '@skyux/i18n';
+
+import { SkyFormsResourcesModule } from '../shared/sky-forms-resources.module';
+
+import { SkyErrorComponent } from './error.component';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-errors',
+  standalone: true,
+  imports: [
+    CommonModule,
+    SkyI18nModule,
+    SkyIdModule,
+    SkyErrorComponent,
+    SkyFormsResourcesModule,
+  ],
+  templateUrl: './errors.component.html',
+  styleUrls: ['./errors.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyErrorsComponent {
+  @HostBinding('attr.aria-live')
+  public readonly ariaLive = 'polite';
```

Comment:
Take a look at this page: https://hidde.blog/how-to-make-inline-error-messages-accessible/#heading-3 and this code example: https://codepen.io/hidde/pen/eRGwer.

They're using `aria-describedby` to link the error region with the input field. Do we also need to do that?

Reply:
I've added `errorId` to the `aria-describedby` — we also apply it to `aria-errormessage`, which seems more appropriate but also does not seem to have much browser support.

Reply:
Like we talked about, we're going to hold off on `aria-describedby` for now.